sdk/storage/azure-storage-blobs/CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 # Release History
 
-## 12.0.0-beta.12 (Unreleased)
+## 12.0.0 (2021-06-08)
```

Comment:
`-08` or `-07`?

Reply:
I think `08` is the right shipping date.